be/src/storage/metadata_util.cpp

```diff
@@ -223,6 +223,23 @@ Status convert_t_schema_to_pb_schema(const TTabletSchema& tablet_schema, uint32_
     }
     schema->set_compress_kind(COMPRESS_LZ4);
 
+    switch (compression_type) {
+    case TCompressionType::LZ4_FRAME:
+        schema->set_compression_type(LZ4_FRAME);
+        break;
+    case TCompressionType::LZ4:
+        schema->set_compression_type(LZ4_FRAME);
+        break;
+    case TCompressionType::ZLIB:
+        schema->set_compression_type(ZLIB);
+        break;
+    case TCompressionType::ZSTD:
+        schema->set_compression_type(ZSTD);
+        break;
+    default:
+        CHECK(false) << "unexpected compression type " << compression_type;
```

Comment:
NOT use CHECK, this will cause the process crash. Should return error for this per request problem

Reply:
OK,  I will fix it.